lib/sisimai/bite/email/facebook.rb

```diff
@@ -0,0 +1,274 @@
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Facebook.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/\AFacebook [<]mailer-daemon[@]mx[.]facebook[.]com[>]\z/,
+        :subject => %r/\ASorry, your message could not be delivered\z/,
+      }
+      Re1 = {
+        :begin   => %r/\AThis message was created automatically by Facebook[.]\z/,
+        :rfc822  => %r/\AContent-Disposition: inline\z/,
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+
+      # http://postmaster.facebook.com/response_codes
+      # NOT TESTD EXCEPT RCP-P2
+      ReFailure = {
+        userunknown: [
+          'RCP-P1', # The attempted recipient address does not exist.
+          'INT-P1', # The attempted recipient address does not exist.
+          'INT-P3', # The attempted recpient group address does not exist.
+          'INT-P4', # The attempted recipient address does not exist.
+        ],
+        filtered: [
+          'RCP-P2', # The attempted recipient's preferences prevent messages from being delivered.
+          'RCP-P3', # The attempted recipient's privacy settings blocked the delivery.
+        ],
+        mesgtoobig: [
+          'MSG-P1', # The message exceeds Facebook's maximum allowed size.
+          'INT-P2', # The message exceeds Facebook's maximum allowed size.
+        ],
+        contenterror: [
+          'MSG-P2', # The message contains an attachment type that Facebook does not accept.
+          'MSG-P3', # The message contains multiple instances of a header field that can only be present once. Please see RFC 5322, section 3.6 for more information
+          'POL-P6', # The message contains a url that has been blocked by Facebook.
+          'POL-P7', # The message does not comply with Facebook's abuse policies and will not be accepted.
+        ],
+        securityerror: [
+          'POL-P1', # Your mail server's IP Address is listed on the Spamhaus PBL.
+          'POL-P2', # Facebook will no longer accept mail from your mail server's IP Address.
+          'POL-P5', # The message contains a virus.
+          'POL-P7', # The message does not comply with Facebook's Domain Authentication requirements.
+        ],
+        notaccept: [
+          'POL-P3', # Facebook is not accepting messages from your mail server. This will persist for 4 to 8 hours.
+          'POL-P4', # Facebook is not accepting messages from your mail server. This will persist for 24 to 48 hours.
+          'POL-T1', # Facebook is not accepting messages from your mail server, but they may be retried later. This will persist for 1 to 2 hours.
+          'POL-T2', # Facebook is not accepting messages from your mail server, but they may be retried later. This will persist for 4 to 8 hours.
+          'POL-T3', # Facebook is not accepting messages from your mail server, but they may be retried later. This will persist for 24 to 48 hours.
```

Comment:
__[rubocop]__
* Metrics/LineLength: Line is too long. [148/144] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->